openreview/conference/helpers.py

```diff
@@ -26,20 +26,20 @@ def get_conference(client, request_form_id):
 
     submission_start_date_str = ''
     submission_start_date = None
-    if note.content.get('Submission Start Date'):
+    if note.content.get('Submission Start Date', '').strip():
```

Comment:
do you mind to assign this to a local variable so we don't need to strip the date several times?